Tests/SafeDIMacrosTests/InstantiableMacroTests.swift

```diff
@@ -626,40 +1023,79 @@ import SafeDICore
                     let receivedB: ReceivedB
                 }
                 """
-            } expansion: {
+            } diagnostics: {
+                """
+                @Instantiable
+                public struct ExampleService: Instantiable {
+                                                           ╰─ 🛑 @Instantiable-decorated type must have a `public` or `open` initializer with a parameter for each @Instantiated, @Received, or @Forwarded-decorated property.
+                                                              ✏️ Add required initializer
+                    public init(forwardedA: ForwardedA, receivedA: ReceivedA) {
+                        self.forwardedA = forwardedA
+                        self.receivedA = receivedA
+                        receivedB = ReceivedB()
+                    }
+
+                    @Forwarded
+                    let forwardedA: ForwardedA
+                    @Received
+                    let receivedA: ReceivedA
+                    @Received
+                    let receivedB: ReceivedB
+                }
+                """
+            } fixes: {
                 """
+                @Instantiable
                 public struct ExampleService: Instantiable {
+                public init(forwardedA: ForwardedA, receivedA: ReceivedA, receivedB: ReceivedB) {
+                self.forwardedA = forwardedA
+                self.receivedA = receivedA
+                self.receivedB = receivedB
+                }
+
                     public init(forwardedA: ForwardedA, receivedA: ReceivedA) {
                         self.forwardedA = forwardedA
                         self.receivedA = receivedA
                         receivedB = ReceivedB()
                     }
```

Comment:
Could this case have the fixit add to the existing init instead of creating a new one?

Reply:
technically yes, but:
1. There are times that this updating an existing `init` won't be what is actually desired.
2. It's a good bit harder to find the closest `init` and suggest updating it. Doable, but harder.
3. Deleting code is easier than writing it.